src/Common/PlatformLinux.inl

```diff
@@ -333,7 +351,12 @@ inline int _filelength(int fd)
 #define _read read
 #define _set_new_handler std::set_new_handler
 #define _finite isfinite
-#define _mkdir(dir) mkdir(dir, S_IRWXU)
+inline int _mkdir(const char *dir)
+{
+    while (char* sep = strchr((char *)dir, '\\')) *sep = '/';
```

Comment:
Please consider moving this code (slashes conversion) to separate function (named like ConvertGamePathToPlatformSpecificPath) with platform-specific implementation. It makes the code more clear and allows to reduce code duplication.